### PR TITLE
[opentelemetry][ci-cd] add section for Ansible tower

### PR DESCRIPTION
**Pull request by v1v**
I digested https://www.elastic.co/blog/5-questions-about-ansible-that-elastic-observability-can-answer and created a brief section about using the Ansible OpenTelemetry callback plugin with the Ansible tower.

cc @yohanswanepoel

### CODE REVIEW

**Comment by bmorelli25** [comment]
What's the status of this PR?

**Comment by v1v** [comment]
Let me change the status, somehow I missed this one! 🙏

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 1]
```diff
@@ -422,6 +422,31 @@ All of these features can help you quickly and visually assess your services use
 .ServiceMap view of a Jenkins pipeline execution instrumented with the Ansible plugin
 image::images/ansible-service-map.png[Ansible service map view]
 
+
```
Comment (bmorelli25) [inline docs/en/observability/ci-cd-observability.asciidoc:425]: This should fix the failing doc build

```suggestion
[discrete]
```

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 2]
```diff
@@ -422,6 +422,31 @@ All of these features can help you quickly and visually assess your services use
...
+[[ci-cd-ansible-tower]]
+=== Ansible AWX/Tower
+
+The Ansible OpenTelemetry plugin integration supports Ansible AWX/Tower, if you'd like to enable it please follow the below steps.
```
Comment (bmorelli25) [inline docs/en/observability/ci-cd-observability.asciidoc:429]:
```suggestion
The Ansible OpenTelemetry plugin integration supports Ansible AWX/Tower. To enable it, follow these steps.
```

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 3]
```diff
@@ -422,6 +422,31 @@ All of these features can help you quickly and visually assess your services use
...
+
+AWX requires an Execution Environment with the Ansible and Python packages installed.
+For this we used the https://www.ansible.com/blog/introduction-to-ansible-builder tool to create the container definition.
```
Comment (bmorelli25) [inline docs/en/observability/ci-cd-observability.asciidoc:432]:
```suggestion
You can use the https://www.ansible.com/blog/introduction-to-ansible-builder[Absible Builder CLI tool] to create the container definition.
```

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 4]
```diff
@@ -422,6 +422,31 @@ All of these features can help you quickly and visually assess your services use
...
+
+You then upload the container to an image repository accessible by AWX and define an Execution Environment using the container you created.
```
Comment (bmorelli25) [inline docs/en/observability/ci-cd-observability.asciidoc:434]:
```suggestion
Then upload the container to an image repository accessible by AWX and define an Execution Environment using the container you created.
```

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 5]
```diff
@@ -422,6 +422,31 @@ All of these features can help you quickly and visually assess your services use
...
+
+To inject the environment variables and service details you can use custom credential types and then assign the credentials to the Playbook template.
+This gives you the flexibility to reuse the endpoint details for Elastic APM and also standardize on custom fields for reporting purposes.
```
Comment (bmorelli25) [inline docs/en/observability/ci-cd-observability.asciidoc:437]:
```suggestion
To inject the environment variables and service details, use custom credential types and assign the credentials to the Playbook template.
This gives you the flexibility to reuse the endpoint details for Elastic APM and also standardize on custom fields for reporting purposes.
```

File: docs/en/observability/ci-cd-observability.asciidoc [hunk 6]
```diff
@@ -422,6 +422,31 @@ All of these features can help you quickly and visually assess your services use
...
+[role="screenshot"]
+.An APM Service Endpoint in Ansible AWX/Tower
+image::images/ansible-automation-apm-endpoint.png[APM Service Endpoint in Ansible Tower]
+
+[role="screenshot"]
+.Custom fileds in Ansible AWX/Tower
+image::images/ansible-automation-apm-service-details.png[Custom fields in Ansible Tower]
+
+Finally, https://www.elastic.co/blog/5-questions-about-ansible-that-elastic-observability-can-answer provides a great
+overview of how all those pieces work together.
```
Comment (bmorelli25) [inline docs/en/observability/ci-cd-observability.asciidoc:448]:
```suggestion
Want to learn more? This https://www.elastic.co/blog/5-questions-about-ansible-that-elastic-observability-can-answer[blog post] provides a great
overview of how all of these pieces work together.
```

**Review by bmorelli25 — APPROVED** [review]
Thank you, @v1v! I pushed a few grammar updates and a syntax change that should fix the failing doc build.

**Pull request merged**